src/EditorFeatures/Core/Implementation/Outlining/AbstractOutliningService.cs

```diff
@@ -23,16 +23,40 @@ internal abstract class AbstractOutliningService : IOutliningService
             _triviaOutlinerMap = defaultTriviaOutlinerMap;
         }
 
-        public async Task<IList<OutliningSpan>> GetOutliningSpansAsync(Document document, CancellationToken cancellationToken)
+        /// <summary>
```

Comment:
`<remark>` not summary. That's sure not something useful to appear in signature help. ;-)